core/src/dird/consolidate.cc

```diff
@@ -92,19 +102,9 @@ static inline void StartNewConsolidationJob(JobControlRecord* jcr,
  */
 bool DoConsolidate(JobControlRecord* jcr)
 {
-  char* p;
-  JobResource* job;
-  JobResource* tmpjob;
-  bool retval = true;
-  char* jobids = NULL;
-  time_t now = time(NULL);
-  int32_t fullconsolidations_started = 0;
-  int32_t max_full_consolidations = 0;
-
-  tmpjob = jcr->dir_impl->res.job; /* Memorize job */
-
   // Get Value for MaxFullConsolidations from Consolidation job
-  max_full_consolidations = jcr->dir_impl->res.job->MaxFullConsolidations;
+  int32_t max_full_consolidations
+      = jcr->dir_impl->res.job->MaxFullConsolidations;
```

Comment:
This should be changed to `const` since the rest of the function also takes care to use `const`.